scripts/image.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+set -x
+VERSION=latest
+IMAGE_TAG=ptpconformance
+REPO=quay.io/deliedit
```

Comment:
move  it to https://quay.io/repository/redhat-cne

Reply:
done